src/liballoc/rc.rs

```diff
@@ -708,7 +884,7 @@ impl<T: ?Sized> Rc<T> {
     ///
```

Comment:
The comment above is outdated now.

Reply:
Applied `s/an unsize value/a possibly-unsized value/` there